lib/griddler/email.rb

```diff
@@ -37,11 +37,12 @@ def config
     end
 
     def recipients(type)
-      params[type].to_a.map { |recipient| extract_address(recipient) }
+      params[type].to_a.map { |recipient| extract_address(recipient) }.compact
```

Comment:
`ignores blank email address` tests that `.compact` is removing the nil elements from the final version of the array - are you looking for a more/less specific test?

One thing I noticed is that `compact` returns `nil` if all elements in the array are nil, which may be unexpected.  What do you think about always returning an empty array from this method?